recipes/dpp/all/conanfile.py

```diff
@@ -0,0 +1,80 @@
+import os
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout, CMakeDeps
+from conan.tools.scm import Git
+from conan.tools.files import download, unzip, get
+
+required_conan_version = ">=1.65"
```

Comment:
```suggestion
required_conan_version = ">=1.54"
```

Nothing currently using a newer Conan version